share/man/man7/mitigations.7

```diff
@@ -189,13 +189,13 @@ is a FreeBSD-specific extension to
 .Xr mmap 2 .
 .Dv PROT_MAX
 provides the ability to set the maximum protection of a region allocated by
-.Xr mmap
+.Xr mmap 2
 and later altered by
-.Xr mprotect .
+.Xr mprotect 2 .
 For example, memory allocated originally with an mmap prot argument of
 PROT_MAX(PROT_READ | PROT_WRITE) | PROT_READ
 may be made writable by a future
-.Xr mprotect
+.Xr mprotect 2
```

Comment:
These four are already addressed in https://github.com/freebsd/freebsd-src/pull/1428, please don't make me redo it.

Reply:
If the changes are exactly the same (and they appear to be!), then git will handle it automagically, so there wouldn't be any need to redo anything.